xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -277,7 +277,13 @@
                 #set($class = $xwiki.getDocument($propClassName).getxWikiClass())
                 #set($list = '')
                 #set($list = $class.get($colname).getListValues())
-                <select id="xwiki-livetable-${htmlLiveTableId}-filter-${velocityCount}" name='${colname}'><option value=''>$services.localization.render('platform.livetable.selectAll')</option>
+                <select id="xwiki-livetable-${htmlLiveTableId}-filter-${velocityCount}" name='${colname}'
+                #if("$!colprop.type" == 'multilist')
+                  class='xwiki-livetable-multilist' multiple='multiple'>
+                #else
+                  ><option value=''>$services.localization.render('platform.livetable.selectAll')</option>
+                  <option disabled>────</option>
```

Comment:
Inconsistent use of valueless attributes, 3 lines above you used `disabled='disabled'`.